docs/intro-to-coalton.md

```diff
@@ -4,9 +4,55 @@ Coalton is a statically typed language that is embedded in, and compiles to, Com
 
 This document is aimed toward people who are already familiar with functional programming languages.
 
+## Systems and Packages
+
+Coalton uses ordinary Common Lisp packages (and ASDF systems) to organize code. If you're starting a new project, add `#:coalton` to your ASD's `:depends-on` list.
+
+Unlike Common Lisp, Coalton's standard library is organized as a large collection of packages. For example, string-related functions are in the `#:coalton-library/string` package. Refer to the [Coalton Reference](https://coalton-lang.github.io/reference) for a complete list of standard library packages.
+
+When making a new project, you will want to define a new package and establish your standard library imports. *Do not `:use` the `#:cl` or `#:common-lisp` packages!* Instead, you'll want to, at minimum, `:use #:coalton`:
+
+```lisp
+(defpackage #:my-package
```

Comment:
I'd prefer the examples use `uiop:define-package` rather than `cl:defpackage`, but I know that the outdated ASDF/UIOP versions bundled by SBCL and CCL make this impractical. Still, I think it would be nice to add an aside that `uiop:define-package` is a better abstraction, but that the version exposed out-of-the-box by SBCL and CCL doesn't support `:local-nicknames`, and therefore that users should install a newer ASDF version if able.

Reply:
I understand the desire, but I don't think it makes sense to put in the intro document. We want experience of trying out Coalton to be as smooth as possible. Telling people to update asdf first would detract from that.

Reply:
I think you're probably right, but it bums me out.